MediaPlayer/tests/MediaPlayer-specs.js

```diff
@@ -105,4 +114,200 @@ describe('MediaPlayer', () => {
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const repeatButton = screen.getByLabelText('Repeat');
+			const controlsFrame = screen.getByTestId('media-player').children.item(3);
+
+			userEvent.click(repeatButton);
+			userEvent.click(repeatButton);
+
+			expect(controlsFrame).toHaveAttribute('repeat', 'all');
+		});
+
+		test('should have `disabled` attribute when `disabled` is defined', () => {
+			render(
+				<MediaPlayer data-testid="media-player" disabled>
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaSlider = screen.getByRole('slider', {hidden: true});
+			const mediaButtons = screen.getAllByRole('button');
+
+			expect(mediaSlider).toHaveAttribute('disabled');
+			mediaButtons.forEach(button => {
+				expect(button).toHaveAttribute('disabled');
+			});
+		});
+
+		test('should render `currentTime`', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFile} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+
+			const currentTime = screen.getByText('00:00');
+
+			expect(currentTime).toBeInTheDocument();
+		});
+
+		test('should have `paused` class on render', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaSlider = screen.getByRole('slider', {hidden: true});
+
+			expect(mediaSlider).toHaveClass('paused');
+		});
+
+		test('should have `full` class name when `type` is not specified', () => {
+			render(
+				<MediaPlayer data-testid="media-player">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaPlayer = screen.getByTestId('media-player');
+
+			const expected = 'full';
+
+			expect(mediaPlayer).toHaveClass(expected);
+		});
+
+		test('should have `tiny` class name when `type="tiny"`', () => {
+			render(
+				<MediaPlayer data-testid="media-player" type="tiny">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaPlayer = screen.getByTestId('media-player');
+
+			const expected = 'tiny';
+
+			expect(mediaPlayer).toHaveClass(expected);
+		});
+
+		test('should have a `locale` attribute when given one', () => {
+			render(
+				<MediaPlayer data-testid="media-player" locale="ar-SA">
+					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
+				</MediaPlayer>
+			);
+			const mediaPlayer = screen.getByTestId('media-player');
+
+			const expected = 'ar-SA';
+
+			expect(mediaPlayer).toHaveAttribute('locale', expected);
+		});
+	});
```

Comment:
On my opinion, this test does not increase code coverage.
I would replace it with a test that uses a different locale and which brings a visual change to the mediaplayer
Example:

import ilib from 'ilib';

		test('should change the format of the displayed time based on the current `locale`', () => {
			ilib.setLocale('si-LK');
			render(
				<MediaPlayer locale="si-LK">
					{audioFiles.map((audioFile, index) => (<source key={index} src={audioFiles[0]} type="audio/mp3" />))}
				</MediaPlayer>
			);

			const currentTime = screen.getByText('00.00');

			expect(currentTime).toBeInTheDocument();
		});